### PR TITLE
inference near plane option

### PR DESCRIPTION
**Pull request by ethanweber**

### CODE REVIEW

File: nerfstudio/models/splatfacto.py [hunk 1]
```diff
@@ -764,6 +766,16 @@ def get_outputs(self, camera: Cameras) -> Dict[str, Union[torch.Tensor, List]]:
         else:
             rgbs = torch.sigmoid(colors_crop[:, 0, :])
 
+        if not self.training and self.config.inference_near_plane != 0.0:
```
Comment (kerrj) [inline nerfstudio/models/splatfacto.py:769]: any update on doing this with the gsplat `project_gaussians` flag instead of manual masks?
